src/Test.js

```diff
@@ -0,0 +1,5 @@
+function Test() {
+alert('Nu Blyat, prosil zhe po bratski ne trogai a ti svoi mishka navel skotina i takoi klacaesh e ara');
```

Comment:
We should wipe that out, I guess.

Reply:
"This" right now used for test some code when needed and planned for some small feature.

Reply:
I recommend to leave a comment regarding this feature and avoid using rude language then. Also please move that to a separate file to avoid multiple components in the same file.

Reply:
Excluded rude language. It's already in a separate file from the start.